src/backend/aspen/test_infra/models/pathogen.py

```diff
@@ -1,9 +1,30 @@
+from typing import Optional
+
 import sqlalchemy as sa
 
 from aspen.database.models import Pathogen
+from aspen.database.models.usergroup import generate_random_id
+
+
+# TODO this should be the functionality we use everywhere.
+def random_pathogen_factory(
```

Comment:
Once we update our tests to include pathogen slugs as part of request URL's, we're going to replace `pathogen_factory` with this method, and fix any reference to this method's old name. We need to make sure our app can support arbitrary pathogen names so this random generator should make that pretty easy.